### ISSUE TITLE
Switch the action to using Zeek's official Docker images

### ISSUE THREAD

**Issue by ckreibich** [zeek/action-zkg-install#7]
The action dates back to when installing our binary packages was the only quick option to get Zeek up and running. Nowadays it'd be nicer to run the action from Zeek's official Docker images, not least because that'll allow people to select specific Zeek versions.

This brings up the question of how to establish a build environment in the image again, so this can support plugins. We could do nothing, leaving it to the user to specify `pkg_sysdeps`, but that's not very nice. We could finally add a helper script to the image, but even if we do that, it'd be good if the action could fall back to installing it itself, in case the Docker image doesn't yet provide that script.  Also, it'd be ideal of this only happens when the Zeek package under test needs it (i.e., when it has a `build_command`).

**Comment by J-Gras** [comment]
Did this make it on any roadmap?

**Comment by bbannier** [comment]
> Did this make it on any roadmap?

No. The [zeek-8.0 project dashboard](https://github.com/orgs/zeek/projects/22/views/1) is public, and in issues in there you would see a link in _Projects_ in the right sidebar.
